post.html

```diff
@@ -0,0 +1,363 @@
+          rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display?styles=98774,98773" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono?styles=36554" rel="stylesheet">
+    <link rel="stylesheet" href="./src/sass/style.css">
+    <title>Nordic Rose</title>
+</head>
+<body>
+<header class="header">
+    <div class="container header__container">
+        <div class="header__logo">
+            <a class="logo__ref" href="#">
+                <img class="logo__img" src="./src/image/svg/nordic_rose_logo_dark.svg" alt="logo">
+            </a>
+        </div>
+        <div class="header__hamburger">
+            <div class="hamburger" id="hamburger">
+                <span class="hamburger__line"></span>
+                <span class="hamburger__line"></span>
+                <span class="hamburger__line"></span>
+            </div>
+        </div>
+        <nav class="header__navigation">
+            <ul class="navigation">
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#">
+                        Blog
+                    </a>
+                </li>
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#about">
+                        About
+                    </a>
+                </li>
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#links">
+                        Links
+                    </a>
+                </li>
+                <li class="navigation__item">
+                    <a class="navigation__link" href="#">
+                        Projects
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </div>
+</header>
+<main class="main">
+    <section id="about" class="about">
+        <div class="container about__container">
+            <h1 class="about__title">
+                A few words about this blog platform, Ghost, and how this site was made
+            </h1>
+            <p class="about__subtitle">
+                Why Ghost (& Figma) instead of Medium, WordPress or other options?
+            </p>
+        </div>
+        <img class="about__image" src="./src/image/svg/about_image.svg" alt="about">
+    </section>
+    <section class="post">
+        <div class="author__wrapper_header">
+            <div class="author">
+                <img class="author__img_header" src="./src/image/jpg/author.jpg" alt="author" width="56" height="56">
+                <div class="author__about">
+                    <p class="author__name_header">
+                        Mika Matikainen
+                    </p>
+                    <p class="author__description_header">
+                        Apr 15, 2020 · 4 min read
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="header-table_wrapper">
+            <table class="social-table">
```

Comment:
I would say that semantically it's not ideal to use table here, but since it works I guess it's ok